modules/installing-microshift-rpm.adoc

```diff
@@ -0,0 +1,28 @@
+= Installing MicroShift
+
+Perform the following steps to install MicroShift:
+
+1. Enable the MicroShift repositories:
+
+    sudo subscription-manager repos \
+        --enable rhocp-4.12-for-rhel-8-$(uname -i)-rpms \
+        --enable fast-datapath-for-rhel-8-$(uname -i)-rpms
+
+2. Install MicroShift:
+
+    sudo dnf install -y microshift
+
+3. Download your installation https://console.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat Hybrid Cloud Console]. This pull secret allows you to authenticate with the container registries that serve the container images used by MicroShift.
+
+4. Copy that pull secret to your RHEL machine at `/etc/crio/openshift-pull-secret` and make it read/write-able by the root user only:
+
+    sudo chown root. /etc/crio/openshift-pull-secret
```

Comment:
```suggestion
    sudo chown root /etc/crio/openshift-pull-secret
```

Reply:
"root." also changes the group, so it may be OK.

Reply:
It may be not be the best known idiom, so will make it explicit as `root:root`.